test/src/sender/main.cpp

```diff
@@ -27,48 +27,65 @@ using TIME = NDTime;
 
 
 /***** SETING INPUT PORTS FOR COUPLEDs *****/
-struct inp_controll : public cadmium::in_port<Message_t>{};
-struct inp_ack : public cadmium::in_port<Message_t>{};
+struct input_control : public cadmium::in_port<Message_t>{};
+struct input_ack : public cadmium::in_port<Message_t>{};
 
 /***** SETING OUTPUT PORTS FOR COUPLEDs *****/
-struct outp_ack : public cadmium::out_port<Message_t>{};
-struct outp_data : public cadmium::out_port<Message_t>{};
-struct outp_pack : public cadmium::out_port<Message_t>{};
+struct output_ack : public cadmium::out_port<Message_t>{};
```

Comment:
change Message_t to message_t, as per naming convention
Note: you may have have to change the name in message.hpp as well

Reply:
Message_t has been changed after review